alibi_detect/od/__init__.py

```diff
@@ -4,6 +4,49 @@
 from .mahalanobis import Mahalanobis
 from .sr import SpectralResidual
 
+from alibi_detect.od.base import TransformProtocol, transform_protocols
```

Comment:
Since this file defines our public API, I would encourage making all utilities private so they don't show up as potential imports at all. E.g. `get_normalizer -> _get_normalizer` and even `from alibi_detect.od.base import TransformProtocol as _TransformProtocol` etc.

Reply:
Maybe the utilities are better suited for `od.base`?

Reply:
I've chosen to move the utilities to `od.base`.